### PR TITLE
Update recommended node instance type and cost

### PR DESCRIPTION
**Pull request by janewang**

### CODE REVIEW

**Comment by janewang** [comment]
How much is `m5-2xlarge 8 core 32GB` @rtsai123 on AWS, thanks

**Comment by rtsai123** [comment]
> 
327.94

**Comment by janewang** [comment]
PR depends on https://github.com/near/near-ops/issues/789  to finalize recommended instance type and cost

File: docs/develop/node/archival/hardware-archival.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Estimated monthly costs depending on operating system:
 | Cloud Provider | Machine Size    | Linux                     |
 | -------------- | --------------- | ------------------------  |
 | AWS            | c5.2xlarge      | $250 CPU + $600 storage † |
-| GCP            | c2-standard-8   | $220 CPU + $600 storage † |
+| GCP            | n2-standard-8   | $380 CPU + $600 storage † |
```
[the remark this answers is not in the document]
Comment (janewang) [reply docs/develop/node/archival/hardware-archival.md:39]: Based on the latest cost, it is ~$280.

**Comment by janewang** [comment]
@nikurt What's the equivalent instance type / cost on Azure? Thanks

**Comment by nikurt** [comment]
Before changing the recommended hardware, I suggest to first test it internally.

**Pull request closed without merging**